server/protocols/protocols-smtp/src/test/java/org/apache/james/smtpserver/ValidRcptHandlerTest.java

```diff
@@ -46,7 +47,7 @@
 
 public class ValidRcptHandlerTest {
 
-    private static final String VALID_DOMAIN = "localhost";
+    private static final Domain VALID_DOMAIN = Domain.of("localhost");
```

Comment:
maybe Domain.of(localhost) could be a constant in Domain?

Reply:
done in another commit